configs/backend_adapter_vendor/ListTemplate.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+//import "../mixer/TemplateVariety.proto";
+import "mixer/configs/mixer/TemplateExtensions.proto";
+
+package istio.mixer.v1.config.template;
+
+message ListTemplate {
+  option (istio.mixer.v1.config.template.template_variety) = TEMPLATE_VARIETY_CHECK;
+  option (istio.mixer.v1.config.template.template_name) = "ListTemplate";
```

Comment:
So, if we make this full message name, then it will have the name of the repo it belongs to. example: mycompany.org.mymixertempaltes.v2.MyMetric Right ?
That would mean all reference to it will have to have that company's name in the cnfg file.
Not sure if that is a concern.

I was thinking we only allow two namespaces, `global` and myservice (sample myservicefoo.googleapis.com). Hopefully we can validate those namespaces during config push. WDUT ?